music21/stream/iterator.py

```diff
@@ -250,7 +250,20 @@ def __getattr__(self, attr):
         >>> s.notes.asdf
         Traceback (most recent call last):
         AttributeError: 'StreamIterator' object has no attribute 'asdf'
+
+        OMIT_FROM_DOCS
+
+        srcStream is accessible, but not with "__getattr__", which joblib uses
+
+        >>> s.notes.srcStream is s
+        True
+        >>> s.notes.__getattr__('srcStream') is None
+        True
         '''
+        # Prevent infinite loop in feature extractor task serialization
+        if attr == 'srcStream':
+            return None
```

Comment:
good catch.  This might cause other bugs along the way, but it's a nice fix for now.

Reply:
I didn't have much of a choice, because it was blowing up otherwise. I think it was the call to coreElementsChanged I removed from stripTies. I don't know how that exposed more functionality (or caused the serialization to happen differently).